core/ulog/log.go

```diff
@@ -60,21 +60,45 @@ type baselogger struct {
 type Log interface {
 	// Configure Log object with the provided log.Configuration
 	Configure(Configuration)
```

Comment:
Yes. I will add a follow up on moving the methods out. I agree that Configuration and SetLogger can be untied from Log interface, but SetLevel pretty much needs a calling logger. If we are not using it, I don't have a strong case of having SetLevel functionality on logger.